bootstrap/shell/shell.go

```diff
@@ -461,6 +461,34 @@ type executeFlags struct {
 	PTY bool
 }
 
+func round(d time.Duration) time.Duration {
+	// The idea here is to show 5 significant digits worth of time.
+	// If your build takes 2 hours, you probably don't care about the timing
+	// being reported down to the microsecond.
+	switch {
+	case d < 100*time.Microsecond:
+		return d
+	case d < time.Millisecond:
+		return d.Round(time.Microsecond / 100)
```

Comment:
[nit]
could we change these round calls in general to:
```suggestion
		return d.Round(time.Nanosecond * 10)
```

and its equivalents for the other switch branches -  IMO it's a lot easier to reason about "round to the nearest 10 nanoseconds" rather than "round to the nearest microsecond / 100

Reply:
Done